app/views/finders/_results.mustache

```diff
@@ -39,4 +39,11 @@
   {{/documents}}
 </ul>
 
+{{#zero_results}}
+  <div class='no-results'>
+    <h2 class='gem-c-heading'>There are no matching results.</h2>
```

Comment:
Obviously this has been merged now, and this is a mustache template so you couldn't have pulled in a component, but we should try to avoid using component classes outside of a component. It would be better to use some styles from GOV.UK Frontend (for example https://design-system.service.gov.uk/styles/typography/) as that gets pulled in with the components gem.